client/delegations_test.go

```diff
@@ -387,6 +392,14 @@ func TestPersistedMeta(t *testing.T) {
 			assert.Equal(t, tt.downloadError, err)
 			assert.Equal(t, tt.fileContent, dest.String())
 
+			target, err := c.Target(tt.file)
+			assert.Equal(t, tt.targetError, err)
+			if tt.targetError == nil {
```

Comment:
What does this do exactly?

Reply:
It ensures that `Target` is able to return delegated targets (without the change, this test would fail)